pom.xml

```diff
@@ -76,10 +76,10 @@ limitations under the License.
     <hamcrest.version>1.3</hamcrest.version>
     <mockito.version>3.6.28</mockito.version>
     <!-- TODO: check if commons-codec was transitively updated to 1.13 and okhttp was updated to 2.7.5 when upgrading-->
-    <beam.version>2.19.0</beam.version>
+    <beam.version>2.24.0</beam.version>
     <!-- referred from bigtable-beam-import and bigtable-emulator -->
     <guava.version>30.0-android</guava.version>
-    <beam-guava.version>20.0</beam-guava.version>
```

Comment:
It seems like beam started vendoring their guava. So I think we should update this prop name to just guava.version